internal/pkg/cli/deploy/workload.go

```diff
@@ -367,7 +378,8 @@ func (d *workloadDeployer) uploadContainerImages(out *UploadArtifactsOutput) err
 	}
 	out.ImageDigests = make(map[string]ContainerImageIdentifier, len(buildArgsPerContainer))
 	for name, buildArgs := range buildArgsPerContainer {
-		digest, err := d.imageBuilderPusher.BuildAndPush(dockerengine.New(exec.NewCmd()), buildArgs)
+		fmt.Print(d.dockerLoginOut)
```

Comment:
probably don't need to print out the login information on every container build - maybe just once before? (if at all; re: my prev comment!)